doc/api/repl.md

```diff
@@ -303,7 +303,20 @@ When a new [`repl.REPLServer`][] is created, a custom evaluation function may be
 provided. This can be used, for instance, to implement fully customized REPL
 applications.
 
-The following illustrates an example of a REPL that squares a given number:
+An evaluation function accepts the following four arguments:
+
+* `code` {string} The code to be executed (e.g. `1 + 1`).
+* `context` {Object} The context in which the code is executed. This can either be the JavaScript `global`
+  context or a context specific to the REPL instance, depending on the `useGlobal` option.
+* `replResourceName` {string} An identifier for the REPL resource associated with the current code
+  evaluation (`REPL<n>`, where `n` is the 1-based index of the current command).
```

Comment:
```suggestion
  The `REPL<n>` might be a bit of an implementation detail and maybe not worth documenting :thinking: 
```

Reply:
yes, I've removed it, it didn't look like an appropriate thing to document :thinking: